types/preact-context-provider.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'preact-context-provider';
```

Comment:
Is `preact-context-provider` for testing purposes?

Reply:
Yes, but it's removed in the next pr(#453).